invoices/views.py

```diff
@@ -140,15 +140,17 @@ def delete_invoice(request, pk):
         if not may_delete_invoice:
             return HttpResponseForbidden('Du har inte behörighet att ta bort denna faktura.')
         invoice.delete()
-        # Inform owner that someone removed it
-        if request.user != invoice.owner.user:
-            receiver_name = invoice.owner.user.first_name + ' ' + invoice.owner.user.last_name
-            deleter_name = request.user.first_name + ' ' + request.user.last_name
-            requests.post("https://spam.datasektionen.se/api/sendmail", json={
-                'from': 'no-reply@datasektionen.se',
-                'to': invoice.owner.user.email,
-                'subject': deleter_name + ' har tagit bort din faktura',
-                'content': render_to_string("remove_invoice_email.html", {'deleter': deleter_name, 'receiver': receiver_name, 'description': invoice.description}),
-                'key': settings.SPAM_API_KEY
-            })
+        if settings.SEND_EMAILS:
+            print("Sent email")
+            # Inform owner that someone removed it
+            if request.user != invoice.owner.user:
+                receiver_name = invoice.owner.user.first_name + ' ' + invoice.owner.user.last_name
+                deleter_name = request.user.first_name + ' ' + request.user.last_name
+                requests.post("https://spam.datasektionen.se/api/sendmail", json={
```

Comment:
Sending emails should probably use the same function since the only real difference is the content and subject. That would also fix that this call should use the `SPAM_URL` env variable.